csun-metalab/profile-manager
# Updated front-end

### Changes
- Pretty much everything in terms of styling
- Made different sections of app responsive
- Styled using Metaphor 2 framework.

## To test
- Click around the app and play with responsiveness to see if nothing gets weird.

## Note
- The META+LAB footer will be added by the end of the week, since I am waiting for an update for Metaphor 2 where it is included.
- Currently using a placeholder image
